CHANGELOG_PENDING.md

```diff
@@ -1,55 +1,24 @@
-# v0.34.0-rc4
+## v0.34.0-rc5
```

Comment:
i think this breaks the linter?

Reply:
Maybe just add a top-level heading then

Reply:
I'll add another headline on top of this, then - it's nice to have the release name at the "##" level of indentation because it matches the actual changelog.

Reply:
Yes, that.